MapboxNavigation/Style.swift

```diff
@@ -38,7 +38,7 @@ open class Style: NSObject {
     /**
      URL of the style to display on the map when previewing a route, for example on CarPlay or your own route preview map.
      */
-    @objc open var previewMapStyleURL = MGLStyle.navigationPreviewDayStyleURL
+    @objc open var previewMapStyleURL: URL = MGLStyle.navigationPreviewDayStyleURL
```

Comment:
Added type explicitly to silence Jazzy warning and correct type displaying in a doc.